### PR TITLE
[DM-39045] Add ssotap

### PR DESCRIPTION
**Pull request by cbanek**

### CODE REVIEW

File: docs/applications/ssotap/index.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+   alert-stream-broker/index
+   exposurelog/index
+   narrativelog/index
+   plot-navigator/index
+   production-tools/index
+   sasquatch/index
+   strimzi/index
+   strimzi-registry-operator/index
+   telegraf/index
+   telegraf-ds/index
```
Comment (jonathansick) [inline docs/applications/ssotap/index.rst:72]: Please delete

File: docs/applications/ssotap/notes.rst [hunk 2]
```diff
@@ -0,0 +1,11 @@
+.. px-app-notes:: ssotap
```
Comment (jonathansick) [inline docs/applications/ssotap/notes.rst:1]: Delete this file

File: docs/applications/ssotap/portal-tap.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
+from diagrams import Cluster, Edge
```
Comment (jonathansick) [inline docs/applications/ssotap/portal-tap.py:1]: Delete this file

Comment (cbanek) [reply docs/applications/ssotap/portal-tap.py]: Okay done

File: docs/applications/ssotap/notes.rst (hunk 2 above)
Comment (cbanek) [reply docs/applications/ssotap/notes.rst]: Okay done

File: docs/applications/ssotap/index.rst (hunk 1 above)
Comment (cbanek) [reply docs/applications/ssotap/index.rst]: Okay did that

File: docs/applications/ssotap/index.rst [hunk 4]
```diff
@@ -0,0 +1,26 @@
+.. px-app:: ssotap
+
+#####################################################
+ssotap — IVOA DP03 Solar System Table Access Protocol
+#####################################################
+
+SSOTAP_ (SSO Table Access Protocol) is an IVOA_ service that provides access to the ObsCore table which is hosted on postgres.
```
Comment (jonathansick) [inline docs/applications/ssotap/index.rst:7]:
```suggestion
SSOTAP (SSO Table Access Protocol) is an IVOA_ service that provides access to the ObsCore table which is hosted on postgres.
```

Drop the link, which has no target specified

File: docs/applications/ssotap/index.rst [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+   :file: applications/_summary.rst.jinja
+
+
+Guides
+======
+
+.. toctree::
+
+   notes
+   values
```
Comment (jonathansick) [inline docs/applications/ssotap/index.rst:26]:
```suggestion
.. toctree::

   values
```

**Pull request merged**